### ISSUE TITLE
Unobtrusive validation

### ISSUE THREAD

**Issue by ghost** [play-components/play-jqvalidate#2]
a better way of client-side validation html generation is using custom html tag attributes (Unobtrusive validation) as this:
http://weblogs.asp.net/mikaelsoderstrom/archive/2010/10/06/unobtrusive-validation-in-asp-net-mvc-3.aspx

**Comment by mmurray** [comment]
The module is intended to output the validation rules in HTML5 data attributes. Your form will end up looking something like this: https://gist.github.com/715557 What about that do you consider obtrusive?

**Comment by ghost** [comment]
I mean like this sample, a rendered asp.net mvc 3 item with validation (there is no js injection into code):

```
    <div class="editor-field">
        <input data-val="true" data-val-number="The field CustomerId must be a number." data-val-required="The CustomerId field is required." id="CustomerId" name="CustomerId" type="text" value="" />
        <span class="field-validation-valid" data-valmsg-for="CustomerId" data-valmsg-replace="true"></span>
    </div>
```

**Comment by ghost** [comment]
Sorry it was already like this, thanks for the superb module.

**Comment by mmurray** [comment]
I see what you mean. The form tag does output a &lt;script&gt; and that is pretty obtrusive. It would be easy to move that js logic to the play-jqvalidate.min.js file.

**Comment by mmurray** [comment]
Okay this is implemented, published as 0.2 in the module repository.

**Comment by ghost** [comment]
:) thank you very much
